package.json

```diff
@@ -10,6 +10,7 @@
     "bootstrap": "3.3.7",
     "bootstrap-datepicker": "1.7.1",
     "c3": "0.4.21",
+    "codemirror": "5.34.0",
```

Comment:
To be honest I find this a bit over the top. I. e. adding a new and non-trivial (6 MB source) dependency for this. Noone in their right mind will edit a super-complex JSON in a browser textarea in Cockpit. There are proper editors for that and copy&paste or file upload to get it into that form. IMHO even showing the text area in the first place is a dangerous path towards creating a full web-based kubernetes  resource text editor (we really don't want to go  down this path, this is a stopgap solution). Can we please just drop this commit?

Reply:
We are not going this path. This is just meant for better user experience.
@jniederm @mareklibra opinions?

Reply:
We aim this to look the same as openshift and be used in patternfly in the future. We can discuss this further in here https://github.com/patternfly/patternfly-react/issues/241

Reply:
it is not really up to me, but up to the designers

Reply:
@andreasn @garrett I guess it's up to you to work with other designers involved in this feature to resolve this?